### PR TITLE
Documentation: GSoD—Improve the clarity of new tutorial

### PR DESCRIPTION
**Pull request by activus-d**
This pull request improves the clarity of the new tutorial.

### CODE REVIEW

**Comment by gasman** [comment]
Hi @activus-d - please could you give this PR a more descriptive title? "Implement feedback" isn't too helpful to anyone reviewing the list of open PRs.

**Comment by activus-d** [comment]
@gasman I've provided a more descriptive title. Thank you

File: docs/tutorial/create_contact_page.md [hunk 1]
```diff
@@ -132,6 +132,10 @@ To add contact information to your portfolio site, follow these steps:
 2. Add the necessary data.
 3. Publish your `Form Page`.
 
+```{note}
+If you want to receive your visitors' messages directly in your inbox, there are further configurations you must implement on your site.
```
Comment (cnk) [inline docs/tutorial/create_contact_page.md:136]: Well now that you have mentioned the possibility of getting the forms in my inbox, I want to know how. Is there a link we could send people to? Or does this information need to be added to this document? If so, what do you think about removing this note so we can merge everything else. Then re-add this note along with the configuration information as a separate PR.

**Review by cnk — COMMENTED** [review]
With one exception (a note about form emailing), everything here looks good and I think could be merged.

File: docs/tutorial/create_contact_page.md (hunk 1 above)
Comment (activus-d) [reply docs/tutorial/create_contact_page.md]: @cnk Thank you for your feedback. I've removed the note and will add detailed information or link(s) on how users can receive messages from their Form page directly in a separate pull request.

**Pull request merged**